torch/ao/quantization/fx/backend_config/quantize_handler.py

```diff
@@ -3,8 +3,29 @@
 from .observation_type import ObservationType
 from ..quantization_patterns import QuantizeHandler
 from ..quantization_types import NodePattern
+from ..utils import all_node_args_have_no_tensors
+from torch.fx import Node
 
-def get_quantize_handler_cls(observation_type, dtype_configs):
+
+def _binary_op_observation_type_getter(quantize_handler):
```

Comment:
I think this function should return the number of tensor arguments.  The logic of "if op is torch.add and num_tensor_args is 1, do this" should be visible from the config.

Reply:
yeah sounds good, I can change the config to a map